BUILD.md

```diff
@@ -22,11 +22,28 @@ To compile Cemu, a recent enough compiler and STL with C++20 support is required
 
 For ubuntu and most derivatives:
 
-1) `sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
-2) Run `git clone --recursive https://github.com/cemu-project/Cemu`
-3) `cd Cemu`
-4) `mkdir build && cd build`
-5) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
-6) `ninja`
 
-Build instructions for other distributions will be added in the future!
+`sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
+
+For Arch and most derivatives:
+
+`sudo pacman -S cmake git base-devel ninja nasm linux-headers zip spirv-headers`
+
+Vcpkg provides spirv-headers but for some it's needed in arch it seems. 
+
+Added zip incase the user runs KDE but with gnome zip is included.
+
+vcpkg is not needed to be installed and provided by cemu when git cloning.
+
+For Fedora and most derivatives:
+
+`sudo dnf in cmake ninja-build vcpkg llvm kernel-headers gtk3-devel libsecret-devel libgcrypt-devel systemd-devel freeglut-devel nasm perl-core zlib-devel cubeb-devel`
+
+1) Run `git clone --recursive https://github.com/cemu-project/Cemu`
+2) `cd Cemu`
+3) `mkdir build && cd build`
+4) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -G Ninja`
+
+4.5) To build for gcc `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++ -G Ninja -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja`
```

Comment:
I don't think that `-DCMAKE_MAKE_PROGRAM` is required

Reply:
It’s required

Reply:
From the [documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_MAKE_PROGRAM.html):

> Tool that can launch the native build system.
> ...
> The [`Ninja`](https://cmake.org/cmake/help/latest/generator/Ninja.html#generator:Ninja) generator sets this to `ninja`.

The `Ninja` generator is enabled with `-G Ninja`, so `CMAKE_MAKE_PROGRAM` is already set to `ninja`. Why is it required?

Reply:
This pr is about deps for arch and fedora only please keep on topic thanks

Reply:
Yes, it is, but you've added an unneeded define that wasn't there before, so I was pointing it out.